experimental/addons/minio-operator/README.md

```diff
@@ -0,0 +1,127 @@
+# minio-operator
+
+MinIO is a Kubernetes-native high performance object store with an S3-compatible API. The MinIO Kubernetes Operator supports deploying MinIO Tenants onto private and public cloud infrastructures ("Hybrid" Cloud).
+
+## Install
+
+Add experimental registry
+```shell
+$ vela addon registry add experimental --type=helm --endpoint=https://addons.kubevela.net/experimental/
+```
+
+Enable this addon
+```shell
+$ vela addon enable minio-operator
+```
+
+```shell
+$ vela ls -A | grep minio
+default         tenant-sample           minio-tenant                            minio-tenant                                    rendering       healthy
+vela-system     addon-minio-operator    ns-minio-operator                       k8s-objects                                     running         healthy
+vela-system     └─                      minio-operator                          helm                                            running         healthy Fetch repository successfully, Create helm release
+```
+
+Disable this addon
+```shell
+$ vela addon disable minio-operator
+```
+
+## Use minio-operator
+
+### Access the Operator Console to interact with tenant.
+
+Run the following command to create a local proxy to the MinIO Operator Console:
+
+```shell
+# Get JWT Token to access the console
+$ kubectl get secret -n minio-operator console-sa-secret -o=jsonpath='{.data.token}' | base64 --decode
+eyJhbGciOiJSUzI1NiIsImtpZCI6ImZoSFNZMFdzUEt3WWpFVFFJdEZwSHZ1ZG9qN1ZwSy1vNC1WUi04b2tpODQifQ.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJtaW5pby1vcGVyYXRvciIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VjcmV0Lm5hbWUiOiJjb25zb2xlLXNhLXNlY3JldCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50Lm5hbWUiOiJjb25zb2xlLXNhIiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZXJ2aWNlLWFjY291bnQudWlkIjoiYTU3YWZmNmQtZTM5Ni00MGE3LTk0NTAtOTc4OTRkNGViY2MyIiwic3ViIjoic3lzdGVtOnNlcnZpY2VhY2NvdW50Om1pbmlvLW9wZXJhdG9yOmNvbnNvbGUtc2EifQ.h3JageNqfiWBjvMZo2XuDvEIZPwZDh2FBu_R9yAiCM248Lso7h1VVorr3yICInOnk5WcpkX6vbY3iTL-VIuNan4Ehx0WDnup4ctnbOjxGin5TJdJuuQKeIVX5adwVDd7jDPD_Pn6YGVbF2kA4538vFQNjauMw3ykmusyQINcjfq1KudwZRvw4ZCLeUVX0MLIVydQDYc2u6J5pX7OwvCUvyIgd711T9T6hLSdr9Hbp9NqrrqaWizfIlhAKlbdv202QG0U3W--BCj81TrPMZdgvF6N-djavGKB0hGlxBiyUeYJ5ncpzdnFD9ncHzj5y7giKCRBp91ctE1pOWLLFcglPA
+
```

Comment:
Please remove any sensitive or confidential information from the documents as it is not considered professional to include them in public documents.

Reply:
Done 👍🏽